bin/manuscripts

```diff
@@ -36,12 +36,15 @@ from dateutil import parser
 sys.path.insert(0, '.')
 from manuscripts.report import Report
 from manuscripts.config import Config
+from manuscripts._version import __version__
 
 def get_params():
     """Parse command line arguments"""
 
     parser = argparse.ArgumentParser()
 
+    parser.add_argument('-v', '--version', action='version',version=__version__,
+                        help="show verson")
```

Comment:
Plese uppercase the sentence, and fix spelling ("Show version")

Reply:
Done